src/UI/Seller/src/app/buyers/components/users/buyer-user-edit/buyer-user-edit.component.ts

```diff
@@ -44,6 +45,8 @@ export class BuyerUserEditComponent {
       LastName: new FormControl(user.LastName, Validators.required),
       Email: new FormControl(user.Email, [Validators.required, ValidateEmail]),
       Country: new FormControl(user.xp?.Country, Validators.required),
+      BuyerGroupAssignments: new FormControl(undefined, this.isCreatingNew ? [Validators.required] : undefined),
+      PermissionGroupAssignments: new FormControl(undefined)
```

Comment:
with undefined, does that still return a FormControl with all the normal operators with just an undefined value?

Reply:
Yes. Alternatively I could use null as well if that is an easier value to work with

Reply:
Undefined is fine, just validating it didn't make the whole control undefined. thanks!